tests/base.py

```diff
@@ -99,6 +100,12 @@ def setUp(self):
             resource = Resource(name='Markers',
                                 quantity=3)
             resource.save()
+            resource_2 = Resource(name='Pen',
```

Comment:
@rajeman You don't need to add a new resource, just use the one already added. Also, you can choose not to assign a resource here, just call the `assign resource` assert method in your test.

Reply:
@ThaDeveloper That doesn't work. If I use the one already added, the test to assign a resource to a room will fail.

Reply:
@rajeman I had made the changes above and all the tests passed.

Reply:
@ThaDeveloper Can we have a sync?

Reply:
@ThaDeveloper I don't think it is a good idea to have what I am not testing for in my test file. I had sync with @jesseinit and we found out that he would also need to copy that test case into his own file if we are to use the method you are suggesting.

Reply:
@rajeman Let's sync on this

Reply:
@ThaDeveloper Kindly check the changes I made.